examples/apps/color_finder.py

```diff
@@ -0,0 +1,740 @@
+        self.pixel_matrix.fill_gaps()
+
+    def approximate_color_of_pixel(self, r, g, b):
+        '''Returns the approximated color of the RGB value of a pixel.
+
+        Args:
+            r (int): the amount of red in the pixel
+            g (int): the amount of green in the pixel
+            b (int): the amount of blue in the pixel
+
+        Returns:
+            closest_color (string): the name of the color closest to the input RGB color
+
+        '''
+        min_distance = sys.maxsize
+        closest_color = ''
+        for color_name, color_values in color_dict.items():
+            d = color_distance_sqr((r, g, b), color_values)
+            if d < min_distance:
+                min_distance = d
+                closest_color = color_name
+        return closest_color
+
+    def get_low_res_view(self):
+        '''Downsizes Cozmo's camera view to the specified dimensions.
+
+        Returns:
+            downsized_image (PIL image): low-resolution version of Cozmo's camera view.
+
+        '''
+        image = color_balance(self.robot.world.latest_image.raw_image)
+        downsized_image = image.resize((self.downsize_width, self.downsize_height), resample = Image.LANCZOS)
+        return downsized_image
+
+    def on_finding_a_blob(self, blob_info):
+        '''Determines whether Cozmo should continue to look at the blob, or drive towards it.
+            
+        Args:
+            blob_info (int, int): coordinates of the blob's center in self.pixel_matrix
+
+        '''
+        self.robot.set_center_backpack_lights(map_color_to_light[self.color_to_find])
+
+        x, y = blob_info
+        WM = 10.0*32/self.downsize_width  # WM and HM are multipliers that adjust the amount to turn
+        HM = 10.0*24/self.downsize_height # based on the current dimensions of self.pixel_matrix.
+        # 'fov' stands for 'field of view'. This is the angle amount
+        # that Cozmo can see to the edges of his camera view.
+        amount_move_head = radians(self.fov_y.radians*((CAMERA_HEIGHT/2)-y*HM)/CAMERA_HEIGHT)
+        amount_rotate = radians(self.fov_x.radians*((CAMERA_WIDTH/2)-x*WM)/CAMERA_WIDTH)
+        if self.moved_too_far_from_center(amount_move_head, amount_rotate):
+            self.state = FOUND_COLOR_STATE
+        if self.state != DRIVING_STATE:
+            self.turn_toward_color_blob(amount_move_head, amount_rotate)
+        else:
+            self.drive_toward_color_blob()
+
+    def moved_too_far_from_center(self, amount_move_head, amount_rotate):
+        '''Decides whether the center of the blob is too far from the center of Cozmo's view.
+
+        Args:
+            amount_move_head (cozmo.util.Angle): 
+                the perceived vertical distance of the blob from center-screen
+            amount_rotate (cozmo.util.Angle): 
+                the perceived horizontal distance of the blob from center-screen
+
+        Returns:
+            too_far (bool): whether the object is too far from center-screen
```

Comment:
Returns should have no var name, like:
bool specifying whether the object is too far from center-screen

Applies to all Returns in the file.